packages/ndla-ui/src/LearningPaths/LearningPathMenu.tsx

```diff
@@ -0,0 +1,394 @@
+  ${fonts.sizes(18, 1.1)};
+  line-height: 20px;
+  margin: ${spacing.small} 0 ${spacing.normal};
+`;
+
+type StepProps = {
+  url: string;
+  title: {
+    title: string;
+  },
+  type: string;
+  id: string | number;
+  current?: boolean;
+}
+
+interface Props {
+  learningsteps: StepProps[];
+  stepId: string | number;
+  name: string;
+  duration: number;
+  lastUpdated: string[];
+  language: string;
+  copyright: {
+    contributors: {
+      type: string;
+      name: string;
+    }[],
+    license: {
+      license: string;
+      description: string;
+      url: string;
+    },
+  };
+  learningPathURL: string;
+  cookies: {
+    [key: string]: string;
+  };
+  t: any;
+}
+
+type renderMenuProps = {
+  learningsteps: StepProps[];
+  isOpen: boolean;
+  currentIndex: number;
+  cookies: {
+    [key: string]: string;
+  };
+}
+
+const renderMenu = ({ learningsteps, currentIndex, isOpen, cookies }:renderMenuProps) => (
+  <nav css={[navCSS, !isOpen && navCSSClosed]}>
+    <ul>
+      {learningsteps.map(({ id, url, title, type }:StepProps, index:number) => {
+        let iconType = type;
+        if (index === currentIndex) {
+          iconType = 'CURRENT';
+        } else if (cookies[id]) {
+          iconType = 'HAS_READ';
+        }
+        return (
+          <StyledMenuItem
+            key={id}
+            current={index === currentIndex}
+            afterCurrent={index > currentIndex}
+            isOpen={isOpen}
+            indexNumber={index}>
+            <SafeLink to={url}>
+              <div css={ContentTypeCSS}>
+                <LearningPathIcon type={iconType} current={index === currentIndex} beforeCurrent={index <= currentIndex} />
+              </div>
+              <span>
+                <span>{title.title}</span>
+              </span>
+            </SafeLink>
+          </StyledMenuItem>
+        );
+      })}
+    </ul>
+  </nav>
+);
+
+const LearningPathMenu: React.FunctionComponent<Props> = ({
+  learningsteps, name, duration, lastUpdated, copyright, stepId, learningPathURL, cookies, t,
+}) => {
+  const [isOpen, toggleOpenState] = useState(false);
+  const { innerWidth } = useWindowSize(100);
+  const currentIndex = learningsteps.findIndex(learningStep => learningStep.id === stepId);
+
+  return (
+    <StyledMenu isOpen={isOpen}>
+      <LearningPathMenuModalWrapper
+        innerWidth={innerWidth}
+        currentIndex={currentIndex}
+        learningstepsTotal={learningsteps.length}
+        closeLabel={t('modal.closeModal')}
+        outOfLabel={t('learningPath.pageOf')}
+      >
+        <div css={css`padding-left: ${spacing.small};`}>
+          <Tooltip align="right" tooltip="åpne">
+            <StyledToggleMenubutton type="button" onClick={() => toggleOpenState(!isOpen)}>
+              {!isOpen ? <ArrowExpandRight /> : <ArrowExpandLeft />}
+            </StyledToggleMenubutton>
+          </Tooltip>
+        </div>
+        <StyledMenuIntro isOpen={isOpen}>
+          <div>
```

Comment:
Noen poeng med en div her?

Reply:
Trengte den for at tooltip'en skulle plasseres riktig..